eZ/Publish/Core/Repository/SiteAccessAware/ContentService.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/**
+ * ContentService class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Repository\SiteAccessAware;
+
+use eZ\Publish\API\Repository\ContentService as ContentServiceInterface;
+use eZ\Publish\API\Repository\Values\Content\ContentCreateStruct;
+use eZ\Publish\API\Repository\Values\Content\ContentUpdateStruct;
+use eZ\Publish\API\Repository\Values\Content\ContentMetadataUpdateStruct;
+use eZ\Publish\API\Repository\Values\Content\LocationCreateStruct;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\API\Repository\Values\User\User;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\Repository\Helper\LanguageResolver;
+
+/**
+ * SiteAccess aware implementation of ContentService injecting languages where needed.
+ */
+class ContentService implements ContentServiceInterface
+{
+    /** @var \eZ\Publish\API\Repository\ContentService */
```

Comment:
I think we are using three lines docblocks in kernel, unless that changed

Reply:
For properties it has started to change some time ago 😏🌚